sim/side.ts

```diff
@@ -190,6 +190,7 @@ export class Side {
 				item: pokemon.item,
 				pokeball: pokemon.pokeball,
 			};
+			if (pokemon.moves.some(move => ['frustration', 'return'].includes(move))) entry.happiness = pokemon.happiness;
```

Comment:
```suggestion
			if (pokemon.moves.some(move => ['frustration', 'return'].includes(move))) {
				entry.happiness = pokemon.happiness;
			}
```

Reply:
I'm not _completely_ sure how I feel about this... It probably makes more sense to send the base power the way Hidden Power is, above.

Reply:
In fact, that's how Return/Frustration work in the move menu, too.

Reply:
Good point! I was unsure about this too, which is why I opened a PR for a one line change